commands/betflip.js

```diff
@@ -11,36 +15,42 @@ module.exports = {
     owner_only: false,
     affect_config: true,
     action: function(message, config) {
-        user = update_dabs(message.author, config)
         content = message.content.toLowerCase().split(" ")
         bonus = 1
-        if (content[1] === "all"){
-            amount = user.dabs
-            choice = content[2]
-            bonus = 2
-        } else if (content[2] === "all") {
-            amount = user.dabs
+        if (content.length !== 3){
+            message.channel.send("Needs 2 arguments. Check $help betflip for" +
+                " more details.")
+            return 1
+        }
+
+        if (content[1].startsWith("h") || content[1].startsWith("t")){
             choice = content[1]
-            bonus = 2
-        } else {
-            amount = Number(content[1])
+            amount = content[2]
+            if (content[2] === "all")
+                bonus = 2
+        } else if (content[2].startsWith("h") || content[2].startsWith("t")){
             choice = content[2]
-            if (amount === NaN || Math.floor(amount) != amount || amount < 0) {
-                amount = Number(content[2])
-                choice = content[1]
-                if (amount === NaN || Math.floor(amount) != amount || amount < 0)
-                    return message.channel.send("Invalid amount.")
-            }
-            if (amount > user.dabs)
-                return message.channel.send("You don't have enough dabs.")
+            amount = content[1]
+            if (content[1] === "all")
+                bonus = 2
+        } else {
+            message.channel.send("Missing h[eads] or t[ails]. Check $help " + 
+                "betflip for more details.")
+        }
+        if (amount === NaN && bonus != 2 || Number(amount) <= 0) {
+            message.channel.send("Invalid amount. Input a positive number " + 
+                "or `all`")
+            return amount
         }
-        if (choice === undefined || (!choice.startsWith("h") && !choice.startsWith("t")))
-            return message.channel.send("Invalid choice.")
 
-        user.dabs -= amount
+        paid_dabs = pay_dabs(message.author, config, amount)
+        if (paid_dabs < 0){
+            message.channel.send("Not enough dabs to bet.")
+            return paid_dabs
+        }
         result = Math.floor(Math.random() * 2)
         title = "Coin flip: "
-        coin = "https://raw.githubusercontent.com/aqua-rar/Nice-Senpai/master/makotocoin"
+        coin = "http://localhost:80/resources/makotocoin"
```

Comment:
This won't work either, the correct fix for that was changing the coin link to "https://raw.githubusercontent.com/aqua-rar/Nice-Senpai/master/resources/makotocoin"

I think this was caused a while ago when I sorted everything into a folder but didn't correct this.